app.py

```diff
@@ -16,15 +17,24 @@ def run_game(layout: Layout, game_resources: GameResources) -> Panel:
     game_resources.player.keyboard_input()
     game_resources.draw()
     panel = Panel(game_resources.level.to_string())
-    layout.update(panel)
+    layout["main game"].update(panel)
```

Comment:
We should use main_game instead of "main game" as while it is valid, it isn't really a convention of setting keys
```suggestion
    layout["main_game"].update(panel)
```